k8s/client-deployment.yaml

```diff
@@ -18,8 +18,9 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: client
-          image: gcr.io/spiffe-io/spire-k8s
-          args: ["/bin/bash", "-c", "sleep infinity"]
+          image: gcr.io/spiffe-io/spire-agent:0.8.0
+          command: ["sleep"]
+          args: ["1000000000"]
```

Comment:
The version of spire-agent as a Workload API Client included in `gcr.io/spiffe-io/spire-k8s` was 0.7.3, so I replaced it with `gcr.io/spiffe-io/spire-agent:0.8.0`.

```bash
$ docker run --rm gcr.io/spiffe-io/spire-k8s /opt/spire/bin/spire-agent --version
0.7.3
```

But I hope to update the version of `gcr.io/spiffe-io/spire-k8s`.
Is gcr.io/spiffe-io/spire-k8s's Dockerfile public on github?